libs/core/src/lib/form/form-message/form-message.component.ts

```diff
@@ -12,15 +23,11 @@ export type MessageStates = 'success' | 'error' | 'warning' | 'information';
     encapsulation: ViewEncapsulation.None,
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class FormMessageComponent {
+export class FormMessageComponent implements CssClassBuilder, OnInit, OnChanges {
     /** Type of the message. Can be 'success' | 'error' | 'warning' | 'information' */
```

Comment:
Shouldn't you update the message type? I see we also have `static` and `embedded`.